docs/rst/statistics_backend/get_data.rst

```diff
@@ -0,0 +1,118 @@
+.. include:: ../exports/alias.include
+
+.. _statistics_backend_get_data:
+
+Retrieve statistical data
+-------------------------
+
+.. warning::
+    This feature is currently not supported.
+    It will be implemented on a future release of *Fast DDS Statistics Backend*.
+
+*Fast DDS Statistics Backend* provides two overloads of |get_data-api| to retrieve statistical data of a given
+|DataKind-api| within a time frame.
+This time interval is evenly divided into the specified number of bins, each one with size
+:math:`(t_to - t_from)/(# of bins)`.
+For each of these bins, a new |StatisticsData-api| value is calculated applying the given |StatisticKind-api| to all the
+data points in it.
+The result is a collection of |StatisticsData-api| elements with size equal to the number of specified bins.
+
+Depending on the |DataKind-api|, the data is related to one or two entities, e.g. |FASTDDS_LATENCY-api| measures the
+latency between a write operation on the data writer side and the notification to the user when the data is available on
+reader side, whereas |HEARTBEAT_COUNT-api| contains the amount of sent HEARTBEATs.
+Because of this difference, |get_data-api| can take either one or two |EntityId-api| related to the |DataKind-api| in
+question.
+The following table illustrates the expected inputs depending on the query's |DataKind-api| passed to |get_data-api|:
+
++-------------------------------+------------------------------------+------------------------------------+
```

Comment:
Add a small description of each data kind or an explicit link to the descriptions.

Reply:
Each entry is already a link to the DataKind API, where the description is